public/less/_base.less

```diff
@@ -1,7 +1,7 @@
 @footer-margin: 40px;
 
 body {
-    font-family: "Lato", "Microsoft YaHei", Arial, Helvetica, sans-serif !important;
+    font-family: "Lato", "Segoe UI", "Microsoft YaHei", Arial, Helvetica, sans-serif !important;
```

Comment:
thehowl Member Pending

What are the latin characters affected by this? Does Lato not have pretty much every latin character you can think of?

Reply:
#3237 was actually filed before the switch to `Lato`, but if `Lato` isn't installed the issue remains.

Reply:
I guess the lato fontface was never added. Fair enough, in any case.